governance/votes/Executive vote - January 24, 2022.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Template - [Executive Vote] Temporarily prevent FLAP auctions and MKR Burn - January 24, 2022
+summary: Increase the System Surplus Buffer to 250 million DAI to prevent FLAP auctions until the auction/burn mechanism can be improved.
```

Comment:
Should we include that we're stopping the lerp in the summary?

Reply:
feels like it's covered in the body text, we're pretty explicit about it preventing flaps.